client/components/spinner-button/index.jsx

```diff
@@ -51,7 +51,8 @@ export default React.createClass( {
 				{ loading &&
```

Comment:
While not introduced in this pull request, a few thoughts on this component:

- Was there a design reference for this? I might have expected (and in fact think I've seen in some places in Calypso) that the spinner be contained within the button, not adjacent.
- Accepting `children` as the text to be shown in the button would feel more natural than a `text` prop, though I guess with the duality of the `loadingText` an argument could be made that it's preferable as-is
- Should we have a default for `loadingText`, like `Loading…` ?

These aren't necessarily blockers to this pull request.